ext/fetch/23_request.js

```diff
@@ -317,12 +318,12 @@
       let signal = null;
 
       // 5.
+      let isRelative = false;
       if (typeof input === "string") {
         let parsedURL;
-        let additionalHeaders = [];
         if (input.startsWith("/")) {
           parsedURL = new URL(input, baseURL);
-          additionalHeaders = [["DT_IS_LOCAL_REQUEST", "true"]];
+            isRelative = true;
```

Comment:
```suggestion
          isRelative = true;
```
NIT: minor indentation error